### PR TITLE
Agency component preview

### LINKED ISSUE #223: Select agency component from agency finder
As a requester, in order to make a request to an agency component, I need the agency component's information to appear below the agency finder when selected.

When selecting an agency component in the agency finder, the agency component's information should be displayed below the agency finder.

## Acceptance criteria

- [ ] Selecting an agency component closes the suggestions window
- [ ] Selecting an agency component shows the agency components information including:
  - [ ] Agency name
  - [ ] Agency component name
  - [ ] Agency component mission
  - [ ] Agency component contact information
  - [ ] Median processing times
- [ ] Search copy from https://github.com/18F/beta.foia.gov/issues/217
- [ ] "Start request" button

### PR DESCRIPTION
**Pull request by hbillings**
Closes #223.

screen shot 2017-10-22 at 12 14 36 pm

### CODE REVIEW

File: js/components/agency_component_preview.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import FoiaPersonnel from './foia_personnel';
+import FoiaSubmissionAddress from './foia_submission_address';
+import PrettyUrl from './pretty_url';
+import { AgencyComponent } from '../models';
+
+
+function AgencyComponentPreview({ agencyComponent }) {
+  const description = AgencyComponent.agencyMission(agencyComponent);
+  const requestUrl = `/request/agency-component/${agencyComponent.id}/`;
+  debugger;
```
Comment (adborden) [inline js/components/agency_component_preview.jsx]: We can remove this :)

File: js/components/agency_component_preview.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        { description &&
+          <div>
+            <h4>Agency mission</h4>
+            <p>{description}</p>
+          </div>
+        }
+
+        { agencyComponent.request_time_stats &&
+          <div>
+            <h4>Median processing time (estimates)</h4>
```
Comment (adborden) [inline js/components/agency_component_preview.jsx]: @RyanSibley do we actually want to refer to these as "estimates"? I feel like we want to steer people away from thinking of these as a prediction.

File: js/components/agency_component_preview.jsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            <AgencyComponentProcessingTime
+              requestTimeStats={agencyComponent.request_time_stats}
+            />
+          </div>
+        }
+
+        <h4>
+          The records or information you&apos;re looking for may already be public.
```
Comment (adborden) [inline js/components/agency_component_preview.jsx]: This should be a `’` or `&rsquo;`.

File: js/components/agency_component_preview.jsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        </h4>
+        <p>You can find out by reaching out to the agency or by visiting
+        their <a href={agencyComponent.website.uri}>website</a> or
+        FOIA <a href={agencyComponent.reading_rooms[0].uri}>reading room</a>.</p>
```
Comment (adborden) [inline js/components/agency_component_preview.jsx]: I don't think we can guarantee that every agency component has this data. I think we need to check for `readings_rooms` and `website`. In the AgencyComponent model I think they default to `[]` and `{}`, but we still need to check the `uri` exists or not.

File: www.foia.gov/_layouts/home.html [hunk 5]
```diff
@@ -29,27 +29,7 @@
...
-        </div>
-      </form>
-      <p class="secondary-text"><a>See full list of agencies</a></p>
-      <h3 class="sans">{{ page.section_three.hed }}</h3>
-      {% for col in page.section_three.cols %}
-        <div class="usa-width-one-half">
-          <h4>{{ col.title }}</h4>
-          <p>{{ col.text }}</p>
-        </div>
-      {% endfor %}
```
Comment (adborden) [inline www.foia.gov/_layouts/home.html]: This content should probably move to a react component.

File: js/components/agency_component_preview.jsx (hunk 1 above)
Comment (hbillings) [reply js/components/agency_component_preview.jsx]: derp

File: js/components/agency_component_preview.jsx (hunk 4 above)
Comment (hbillings) [reply js/components/agency_component_preview.jsx]: Cool, I'll implement that as a conditional similar to `request_time_stats` unless there's a different way I should do it!

File: www.foia.gov/_layouts/home.html (hunk 5 above)
Comment (adborden) [reply www.foia.gov/_layouts/home.html]: Scratch that, sounds like this content is being moved up on the landing page.

**Comment by adborden** [comment]
I'm merging some of these PRs that change a few assumptions we have here. a) how we grab the selected agency component from the store and b) the request processing stats.

Grab me when you're ready to merge and I'll help you out with the conflicts.

**Comment by hbillings** [comment]
Default screen with new copy:
screen shot 2017-10-24 at 6 22 14 pm

A search result:
screen shot 2017-10-24 at 6 22 26 pm

**Comment by hbillings** [comment]
@adborden, this is ready for your eyes again.

File: js/models/agency_component.js.orig [hunk 6]
```diff
@@ -0,0 +1,109 @@
+import { List, Map, Record } from 'immutable';
```
Comment (adborden) [inline js/models/agency_component.js.orig]: Oops, this file is left over from the merge, would you please remove it?

File: webpack.config.js.orig [hunk 7]
```diff
@@ -0,0 +1,66 @@
+const assert = require('assert');
```
Comment (adborden) [inline webpack.config.js.orig]: This file is also left over from the merge.

**Pull request merged**